usage/benchmark_async_vs_sync.py

```
@@ -0,0 +1,291 @@
+        f'## 📊 图片下载与解密性能\n\n'
+        f'| 模式 | 平均耗时 | 状态 |\n'
+        f'| :--- | :--- | :--- |\n'
+        f'| Sync  Download | {avgs["sd"]:.4f}s | {"✅" if avgs["sd"] > 0 else "❌"} |\n'
+        f'| Async Download | {avgs["ad"]:.4f}s | {"✅" if avgs["ad"] > 0 else "❌"} |\n\n'
+        f'{perf_line(avgs["sd"], avgs["ad"])}'
+    )
+
+    with open('PERFORMANCE_REPORT.md', 'w', encoding='utf-8') as f:
+        f.write(report)
+
+    print(f'\n✅ 评测完成，报告已生成: PERFORMANCE_REPORT.md')
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)